### DISCUSSION TITLE
After randomly selecting every beatmap more than once, rewind keeps selecting the same beatmap

### DISCUSSION THREAD

**Discussion by LittleEndu** [ppy/osu#25890] [Q&A] [answered]
Was told to open an issue but logs are required and I can't generate ones right now 🤷 

https://github.com/ppy/osu/pull/23312#issuecomment-1859772108

https://github.com/ppy/osu/blob/469a659938dfb3c696c9a9fb51c7252214014a31/osu.Game/Screens/Select/BeatmapCarousel.cs#L623-L624

This used to be a stack, but after changing it to list, rewind now selects from back but removes from front, causing the selection of the same beatmap until every instance of it has been removed

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/25904

logs are not required if the issue is an obvious issue and logs are not bound to help much here anyways

the fact that this is a discussion feels like malicious compliance
